packages/jest-runner/src/testWorker.ts

```diff
@@ -86,7 +86,10 @@ export async function worker({
       globalConfig,
       config,
       getResolver(config, moduleMap),
-      context,
+      context && {
+        ...config,
```

Comment:
`context` not `config`, no?

Reply:
yea, how did that passed typecheck :o

Reply:
I think `config` is a superset of `context`